lighthouse-core/report/html/report-styles.css

```diff
@@ -1366,3 +1366,29 @@
   75% { opacity: 1; }
   100% { opacity: 1;  filter: drop-shadow(1px 0px 1px #aaa) drop-shadow(0px 2px 4px hsla(206, 6%, 25%, 0.15)); pointer-events: auto; }
 }
+
+/* Element screenshot */
+.lh-element-screenshot {
+  position: relative;
+  pointer-events: none;
+}
+.lh-element-screenshot__content {
+  position: absolute;
+  right: 0px;
+  top: -190px;
```

Comment:
this is based on the dynamic height we set it js? anyway to fix it or not have the hanging magic value? :)

Reply:
It's so that the screenshot doesn't cover the table item you're hovering over and instead appears above it. Not really based on anything though.

Do you think showing the screenshot above the table item on hover is the way to go? And then on mobile we could expand the table item to include the screenshot when the user taps on it.